react/features/video-layout/functions.js

```diff
@@ -110,7 +110,9 @@ export function getTileViewGridDimensions(state: Object) {
         - (iAmRecorder ? 1 : 0)
         - (disableSelfView ? 1 : 0);
 
-    const columnsToMaintainASquare = Math.ceil(Math.sqrt(numberOfParticipants));
+    // When there are 3 participants in the call we want them to be placed on a single row unless the maxColumn setting
+    // is lower.
+    const columnsToMaintainASquare = numberOfParticipants === 3 ? 3 : Math.ceil(Math.sqrt(numberOfParticipants));
```

Comment:
Nit: can we make that number 3 a constant?

Reply:
This is mostly an exception. What would be the benefit of having constant?

Reply:
Readability, nothing special.